lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -1401,6 +1408,28 @@ static void maybeDiagnoseClassWithoutInitializers(ClassDecl *classDecl) {
   diagnoseClassWithoutInitializers(classDecl);
 }
 
+void TypeChecker::checkResultType(Type resultType,
```

Comment:
We should do something with this or remove it.

Reply:
ah, yes I'll remove this.